src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
+  postId: number;
+  id: number;
+  name: string;
+  email: string;
+  body: string;
+}
```

Comment:
and an empty line at the end of each file

Reply:
My local file already has an empty line.